### ISSUE TITLE
Worn items need a more human readable format

### ISSUE THREAD

**Issue by Zironic** [Arx-Game/arxcode#53]
Right now when you look at a character their items are printed in an essentially randomly ordered very colorful list. The net effect of this is that it's almost impossible to tell at a glance what a character is wearing as their shirt or earring etc.

Since worn items are attached to a paperdoll system, I think it would make a lot of sense if that paperdoll was printed or at the very least the items are always printed in the same order.

**Comment by TehomCD** [comment]
I remember it being a very early feature that was requested for the item order to be something players could set, via it being ordered based on equip order. I want to say outfits also have some way to control the order, but I don't remember offhand.

This kind of poses the question of whether we want to have the order be adjustable by the viewer with a setting (like letting you always see things based on slot in a consistent order), or always leave presentation up to the person being viewed.

**Comment by rozzingit** [comment]
Yeah, worn items aren't in random order, they show up in the order the PC equips them. And when you make a new outfit, it remembers the order you're wearing items in and it restores them in that specific order when you wear/outfit.

**Comment by Zironic** [comment]
I'd say you want a reasonably sensible default and then allow people to customize it. What some games do is allow you to give items wear strings, so you can have things like
Wrapped around head: Fancy scarf
Piercing left eyebrow: Star metal pin
etc.

**Comment by jereco** [comment]
I've changed my item titles to form a type of description, which relies on them keeping their worn order. 

ex.
> A dark black umbra top clinging to his form ends at, a stylish leather belt holding up, his matching umbra trousers cut cleanly above, a sturdy pair of leather boots,

While this was entertaining to do, it might having this better enabled would be attractive. Though there are often questions about how many of 'x' can be worn where and what is what, etc.

**Comment by Zironic** [comment]
I was looking at the wearable mixin today and I realized that there exist one major problem in doing this in a consistent way. And that is that as far as I can see, there is no actual defined 'paperdoll' which the items attach to, rather they just check if any other item is trying to be in the same 'slot' as another item.
